src/data_sets.h

```diff
@@ -1704,16 +1704,19 @@ typedef struct PACKED
 } sys_sensors_adc_t;
 
 #if defined(IMX_5)
-    #define NUM_COM_PORTS       4    // Number of communication ports.  (Ser0, Ser1, Ser2, and USB).
+    #define NUM_COM_PORTS           4    // Number of communication ports.  (Ser0, Ser1, Ser2, and USB).
 #elif defined(GPX_1)
-    #define NUM_COM_PORTS   6
+    #define NUM_COM_PORTS           6
 #else
-    #define NUM_COM_PORTS   6
+    #define NUM_COM_PORTS           6
 #endif
+
 #ifndef NUM_SERIAL_PORTS
-#define NUM_SERIAL_PORTS    NUM_COM_PORTS
+#define NUM_SERIAL_PORTS        NUM_COM_PORTS
 #endif
 
+#define GPX_NUM_NON_GNSS_PORTS  4
```

Comment:
I'm not really a fan of this variable name.  Super minor, but I can totally imagine this come up again in the future...   Unfortunately, I don't have a better suggestion.